packages/collector/test/tracing/opentelemetry/test.js

```diff
@@ -343,211 +744,113 @@ mochaSuiteFn('opentelemetry/instrumentations', function () {
-        .then(() => retry(() => agentControls.getSpans().then(spans => expect(spans).to.be.empty))));
-  });
+          });
+        });
 
-  describe('socket.io', function () {
-    globalAgent.setUpCleanUpHooks();
-    const agentControls = globalAgent.instance;
-    let socketIOServerPort;
+        describe('opentelemetry is disabled', function () {
+          globalAgent.setUpCleanUpHooks();
+          const agentControls = globalAgent.instance;
+
+          let controls;
+
+          before(async () => {
+            controls = new ProcessControls({
+              appPath: path.join(__dirname, './oracle-app'),
+              useGlobalAgent: true,
+              cwd: __dirname,
+              env: {
+                INSTANA_DISABLE_USE_OPENTELEMETRY: true,
+                OTEL_API_VERSION: version
+              }
+            });
 
-    let serverControls;
-    let clientControls;
+            await controls.startAndWaitForAgentConnection();
+          });
 
-    before(async () => {
-      socketIOServerPort = portfinder();
-
-      serverControls = new ProcessControls({
-        appPath: path.join(__dirname, './socketio-server'),
-        useGlobalAgent: true,
-        env: {
-          SOCKETIOSERVER_PORT: socketIOServerPort
-        }
-      });
+          beforeEach(async () => {
+            await agentControls.clearReceivedTraceData();
+          });
 
-      clientControls = new ProcessControls({
-        appPath: path.join(__dirname, './socketio-client'),
-        useGlobalAgent: true,
-        env: {
-          SOCKETIOSERVER_PORT: socketIOServerPort
-        }
-      });
+          after(async () => {
+            await controls.stop();
+          });
 
-      await clientControls.startAndWaitForAgentConnection();
-      await serverControls.startAndWaitForAgentConnection();
-    });
+          afterEach(async () => {
+            await controls.clearIpcMessages();
+          });
 
-    beforeEach(async () => {
-      await agentControls.clearReceivedTraceData();
+          it('should trace instana spans only', () =>
+            controls
+              .sendRequest({
+                method: 'GET',
+                path: '/trace'
+              })
+              .then(() =>
+                retry(() =>
+                  agentControls.getSpans().then(spans => {
+                    expect(spans.length).to.equal(1);
+
+                    verifyHttpRootEntry({
+                      spans,
+                      apiPath: '/trace',
+                      pid: String(controls.getPid())
+                    });
+                  })
+                )
+              ));
+        });
+      });
     });
+  });
 
-    after(async () => {
-      await serverControls.stop();
-      await clientControls.stop();
+  describe('when otel sdk and instana is enabled', function () {
+    this.timeout(config.getTestTimeout() * 2);
+    before(async () => {
+      execSync('npm install --no-save --no-package-lock', {
```

Comment:
Can you please add a `rm -rf node_modules..` execSync before?
Otherwise its hard to test this - core.tgz and collector.tgz should never be cached locally.
We always want a clean state.